README.md

```diff
@@ -69,7 +69,7 @@ For more information about spotless checks see
     Example configs/store_config.yml:
     ```yaml
     endpointUrl:
-      retrieve: http://localhost:9041/mis/product/
+      retrieve: http://localhost:9041/dataset/
```

Comment:
Now that I am reading this in this PR, it seems a little odd we are calling this dataset, but only sending a single file.

Reply:
Actually, it wouldn't "break" our contract with Transform Service.

https://github.com/connexta/ion-transformation-api/blob/7f51d2ab85fb4684e3c71d47d8931d3f3d1242bb/rest/specs/src/main/filtered-resources/openapi/openapi.yaml#L198-L200

So, if the `file` location was something like  `/datatset/<datasetId>/file`, the cst location would have to be `/datatset/<datasetId>/file/metadata/cst`.